packager/mpd/base/xml/xml_node.cc

```diff
@@ -463,11 +463,36 @@ bool RepresentationXmlNode::AddAudioChannelInfo(const AudioInfo& audio_info) {
     // Convert EC3 channel map into string of hexadecimal digits. Spec: DASH-IF
     // Interoperability Points v3.0 9.2.1.2.
```

Comment:
Can you include the associated spec for the below change?

Reply:
Sure. It comes from ETSI TS 102 366 V1.4.1 Digital Audio Compression (AC-3, Enhanced AC-3), chapter I.1.2.1. I will add the comments.